src/test/java/uk/gov/hmcts/reform/civil/service/docmosis/aos/AcknowledgementOfClaimGeneratorTest.java

```diff
@@ -143,9 +145,37 @@ void shouldGenerateAcknowledgementOfClaim_when1V2SameSolicitorDataIsProvided() {
             .respondent2SameLegalRepresentative(YES)
             .build();
 
+        AcknowledgementOfClaimForm expectedDocmosisData = AcknowledgementOfClaimForm.builder()
```

Comment:
This section of data preparation seems to be duplicated (or very similar) in three tests changed in this PR. Please can we create a private method so as not to repeat this block of code.